### ISSUE TITLE
NullPointerException: Attempt to invoke interface method 'SharedPreferences.getBoolean(java.lang.String, boolean)' on a null object reference

### ISSUE THREAD

**Issue by sarankumar-ns** [zoho/salesiq-mobilisten-flutter#17]
Hi,
I was getting NullPointerException

````
Caused by java.lang.NullPointerException: Attempt to invoke interface method 'boolean android.content.SharedPreferences.getBoolean(java.lang.String, boolean)' on a null object reference
       at com.zoho.livechat.android.utils.LiveChatUtil.isStartChatEnabled(LiveChatUtil.java:7)
       at com.zoho.livechat.android.ui.activities.ChatActivity.initChat(ChatActivity.java:205)
       at com.zoho.livechat.android.ui.activities.ChatActivity.onCreate(ChatActivity.java:159)
       at android.app.Activity.performCreate(Activity.java:8290)
       at android.app.Activity.performCreate(Activity.java:8270)
       at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1329)
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:4009)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4201)
       at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
       at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
       at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2438)
       at android.os.Handler.dispatchMessage(Handler.java:106)
       at android.os.Looper.loopOnce(Looper.java:226)
       at android.os.Looper.loop(Looper.java:313)
       at android.app.ActivityThread.main(ActivityThread.java:8663)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
````

**Comment by DavidBhahirathan** [comment]
Hi @sarankumar-ns  ,

Thank you for writing to us. We'll look into this crash report and update you here once it is fixed. 

Regards,
Bhahirathan M

**Comment by DavidBhahirathan** [comment]
Hi @Ramanpreet6262 ,

The above crash has been fixed in the version [2.0.0](https://github.com/zoho/SalesIQ-Mobilisten-flutter/releases/tag/v2.0.0).

Regards,
Bhahirathan M,
Zoho SalesIQ.
